crates/router/src/types/api/customers.rs

```diff
@@ -20,19 +20,19 @@ pub(crate) trait CustomerRequestExt: Sized {
     fn validate(self) -> RouterResult<Self>;
 }
 
-impl CustomerRequestExt for CustomerRequest {
-    fn validate(self) -> RouterResult<Self> {
-        self.email
-            .as_ref()
-            .validate_opt(|email| utils::validate_email(email.peek()))
-            .change_context(errors::ApiErrorResponse::InvalidDataFormat {
-                field_name: "email".to_string(),
-                expected_format: "valid email address".to_string(),
-            })?;
-
-        Ok(self)
-    }
-}
+// impl CustomerRequestExt for CustomerRequest {
+//     fn validate(self) -> RouterResult<Self> {
+//         self.email
+//             .as_ref()
+//             .validate_opt(|email| utils::validate_email(email.peek()))
+//             .change_context(errors::ApiErrorResponse::InvalidDataFormat {
+//                 field_name: "email".to_string(),
+//                 expected_format: "valid email address".to_string(),
+//             })?;
+//
+//         Ok(self)
+//     }
+// }
```

Comment:
You should be able to remove the commented code, and the `CustomerRequestExt` trait definition.

Reply:
@nachiketkanore Could you remove the commented code?